examples/bootstrap-script/README.md

```diff
@@ -0,0 +1,125 @@
+# Bootstrapping your users
+
+Before spawning a notebook to the user, it could be useful to 
+do some preparation work in a bootstrapping process.
+
+Common use cases are:
+
+*Providing writeable storage for LDAP users*
+
+Your Jupyterhub is configured to use the LDAPAuthenticator and DockerSpawer.
+
+* The user has no file directory on the host since your are using LDAP.
+* When a user has no directory and DockerSpawner wants to mount a volume,
+the spawner will use docker to create a directory.
+Since the docker daemon is running as root, the generated directory for the volume 
+mount will not be writeable by the ```jovyan``` user inside of the container. 
```

Comment:
Future-reference markdown note: You can use single back-ticks:

```markdown
like `this`
```

for inline blocks. You only need triple-backticks for indented code blocks.